docs/dsp-tools-xmlupload.md

```diff
@@ -793,6 +789,100 @@ Example:
 ```
 
 
+## DSP base resources / base properties to be used directly in the XML file
+There is a number of base resources and base properties that must not be subclassed in a project ontology. They are 
+directly available in the XML data file. Please have in mind that built-in names of the knora-base ontology must be used 
+without prepended colon.  
+See also [the related part of the ontology documentation](dsp-tools-create-ontologies.md#dsp-base-resources-base-properties-to-be-used-directly-in-the-xml-file)
+
+### `<annotation>`
+`<annotation>` is an annotation to another resource of any class. It must have the following predefined properties:
+
+- `hasComment` (1-n)
+- `isAnnotationOf` (1)
+
+Example:
+```xml
+<annotation label="Annotation to another resource" id="annotation_0" permissions="res-default">
+    <text-prop name="hasComment">
+        <text encoding="utf8" permissions="prop-default">This is an annotation to a resource.</text>
+    </text-prop>
+    <resptr-prop name="isAnnotationOf">
+        <resptr permissions="prop-default">img_1</resptr>
+    </resptr-prop>
+</annotation>
+```
+
+Technical note: An `<annotation>` is in fact a `<resource restype="Annotation">`.
```

Comment:
isn't this a bit confusing? users might then try to use `<resource restype="Annotation">`